sites/friday/src/manage-resources/resource-init.md

```diff
@@ -115,79 +171,202 @@ platform integration:add \
   --resources-init {{< variable "INITIALIZATION_STRATEGY" >}}
 ```
 
-To specify a resource initialization strategy for an existing source integration,
+To set a resource initialization strategy for an existing source integration,
 run the following command:
 
-```bash
+```bash {location="Terminal"}
 upsun integration:update --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
 ```
 
 For example, to use the `minimum` strategy for your deployment, run the the following command:
 
-```bash
+```bash {location="Terminal"}
 upsun integration:update --resources-init=minimum
 ```
 
 {{< /codetabs >}}
 
-### Minimum resources
+### Environment branch
```

Comment:
Something new with the branch operation => even when using the another strategy than parent (such as `default` or `minimum`, since we removed `manual`), we are initializing the disk size with the same disk as the parent.
This is for the same reasons as when syncing the data from parent to the child. The child needs to have enough disk to clone the data from the parent. So even if the user choose the default or minimum strategy, the disk size will be the one from the parent.

Reply:
Note added at the end of the section.

Reply:
Thanks, the note is great.
We could maybe just put it below the tabs, because at the moment, it only appears in the `Using the CLI tab`, and not in the `In the Console` tab.

Reply:
You're absolutely right, I've moved the note up.